dist/gmusic.js

```diff
@@ -165,6 +165,15 @@ GMusic.Playback = {
   ALL_SHUFFLE: 'ALL_SHUFFLE',
```

Comment:
That's a huge overhaul, I want some more opinions before I start converting that over...